--- repo/elink-gold/version.v
+++ repo/elink-gold/version.v
@@ -26,17 +26,17 @@
 // Version register is {8'Generation, 8'Platform, 8'Type, 8'Version}
 //  Please see versions.txt in the top-level fpga directory.
 // Generation 1 - Parallella-I
-// Platform   1 - E16, 7Z020, GPIO
+// Platform   3   E16, 7Z010, GPIO
 // Type       3 - Headless, 48 singled-ended GPIOs from EMIO
 
 // Version    1 - First E16/Type3 4/22/14, FH
 //            2 - Added cclk gating on e-reset, tx-disable, 5/5/14, FH
 
-`define VERSION_VALUE   {8'h01, 8'd01, 8'd03, 8'd02}
+`define VERSION_VALUE   {8'h01, 8'd03, 8'd03, 8'd02}
 
 // Define one of the following for target FPGA
-//`define TARGET_7Z010 1
-`define TARGET_7Z020 1
+`define TARGET_7Z010 1
+//`define TARGET_7Z020 1
 
 // Define one of the following for target processor
 `define TARGET_E16 1
